spec/controllers/wishlists_controller_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "rails_helper"
+
+RSpec.describe WishlistsController, type: :controller do
```

Comment:
You don't have to specify type. Type is guessed by RSpec and set in config in `spec/rails_helper.rb`.